crates/wallet/src/wallet/mod.rs

```diff
@@ -393,7 +446,7 @@ impl Wallet {
     /// Note that the descriptor secret keys are not persisted to the db. You can either add
     /// signers after-the-fact with [`Wallet::add_signer`] or [`Wallet::set_keymap`]. Or you can
     /// add keys when building the wallet using [`LoadParams::keymap`] and/or
-    /// [`LoadParams::descriptors`].
+    /// [`LoadParams::descriptor`].
```

Comment:
⛏️ Should mention `extract_keys` here:
```suggestion
    /// [`LoadParams::descriptor`] with a private key descriptor and [`LoadParams::extract_keys`].
```

Reply:
@notmandatory fixed